src/mlpack/methods/ann/convolution_rules/svd_convolution.hpp

```diff
@@ -24,7 +24,7 @@ namespace mlpack {
  * Computes the two-dimensional convolution using singular value decomposition.
  * This class allows specification of the type of the border type. The
  * convolution can be computed with the valid border type of the full border
- * type (default).
+ * type (default). Dilation and stride arguments must be equal to one.
```

Comment:
Honestly, I would remove the SVD and FFT convolution entirely from the codebase; initially, they were added for research purposes. And I don't think anyone uses the implementation.